app/views/layouts/application.html.erb

```diff
@@ -2,8 +2,8 @@
 <html>
```

Comment:
You haven't added 'ng-app' and 'ng-view' yet, so the view is still using the default Rails view.